### ISSUE TITLE
USWDS - Bug: Date Picker keyboard navigation not working correctly with JAWS

### ISSUE THREAD

**Issue by ajanickiv** [uswds/uswds#4616]
## Date Picker keyboard navigation not working correctly when using the JAWS screen reader

When using the JAWS screen reader the left and right arrow keys do not work to choose new dates in the date picker calendar pop up.

The left and right arrow keys do work with the date picker calendar pop up when the screen reader is not enabled.

## Steps to reproduce the issue

1. Navigate to the [date picker examples on the USWDS website](https://designsystem.digital.gov/components/date-picker/)
2. Turn on JAWS screen reader
3. Using keyboard or mouse, open calendar popup
4. Attempt to select a new date using the left or right arrow keys

**Comment by brunerae** [comment]
Hey there @ajanickiv! Thanks so much for flagging this; we'll look into it.

**Comment by mejiaj** [comment]
@ajanickiv we had a similar issue pop in the past [here](https://github.com/uswds/uswds/issues/4271#issuecomment-885205949). There were instructions that delayed the interaction on dates.

Are you not able to interact **at all** with the calendar popup?

**Comment by mejiaj** [comment]
Assigning to @brunerae because we need to follow-up with a11y group.

**Comment by brunerae** [comment]
hi @ajanickiv which browser were you on when you experienced this issue? Thanks!

**Comment by brunerae** [comment]
@mejiaj / @thisisdano see below from A11Y testers:

Things we noticed:

- In JAWS Left and right arrows do not work to choose new dates/navigate the calendar table but up and down arrows do. The up and down arrows move the user left and right. 
- I tested this behavior via VoiceOver and Safari and could not replicate this JAWS behavior. Was easily able to move around the calendar. 
- [Other tester] tested this on NVDA and is able to move using left and right errors BUT before moving from date to date, each date was spelled out letter by letter. You can move much more easily with the up and down arrows. 
- We were not able to replicate the second issue. Both [other tester] and I were able to use our arrow keys within the calendar, without a screen reader on. I’m on a Mac and [other tester] is on a PC. 

**Bottom line:** We don’t fully understand why this behavior is happening but suspect the answer can be found in the dates/buttons.

**Comment by amycole501** [comment]
Left and right arrows do not work. The up and down arrows do. 
To navigate to the right, I have to click the down arrow. To navigate to the left date I have to use the up arrow.
From what JAWS says aloud, the right arrow key just starts reading each letter in the top of the calendar. For example, "Space. M. A. Y." While the up and down arrows work great. The issue is there's no way to use the up and down arrows to go up and down. 

incidentally, using Windows narrator, the arrow keys close the calendar entirely so the entire component is useless with a keyboard + narrator.

NVDA- arrow keys do not allow you to navigate the calendar at all but at least it doesn't close it out entirely.
